plugins/filter_aws/aws.c

```diff
@@ -740,6 +928,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_filter_aws, hostname_include),
      "Enable EC2 instance hostname"
     },
+    {
+     FLB_CONFIG_MAP_BOOL, "tags", "false",
+     0, FLB_TRUE, offsetof(struct flb_filter_aws, tags_include),
```

Comment:
since we discussed in the comments about later support an `include_tags` option, let's call the flag on the context something like `tags_enabled`. That won't conflict and also is better for a boolean IMO.

Reply:
Done, https://github.com/fluent/fluent-bit/pull/6065/commits/073cac3c01850100d31fb52cc20e3e703efae39a

Reply:
Conversation resolved due to lack of new activity. (Feel free to bring it back.)